pterm_test.go

```diff
@@ -1,6 +1,11 @@
 package pterm_test
 
 import (
+<<<<<<< HEAD
```

Comment:
Same as the first file, I think some merge conflict Croft just got stuck in here.